src/utils.ts

```diff
@@ -0,0 +1,203 @@
+"use strict";
+
+import Table = require("cli-table");
+import colors = require("colors/safe");
+import { FormatOptions } from "./index";
+import { getDebugger } from './debug';
+
+const debug = getDebugger("utils");
+
+interface TableOptions {
+    chars: Partial<Record<(
+        "top" |
+        "top-mid" |
+        "top-left" |
+        "top-right" |
+        "bottom" |
+        "bottom-mid" |
+        "bottom-left" |
+        "bottom-right" |
+        "left" |
+        "left-mid" |
+        "mid" |
+        "mid-mid" |
+        "right" |
+        "right-mid" |
+        "middle"
+    ), string>>;
+    truncate: string;
+    colors: boolean;
+    colWidths: number[];
+    colAligns: Array<"left" | "middle" | "right">;
+    style: Partial<{
+        "padding-left": number;
+        "padding-right": number;
+        head: string[];
+        border: string[];
+        compact: boolean;
+    }>;
+    head: string[];
+}
+
+/**
+ * @class
+ * @extends Table
+ */
+class StrippedTable extends Table {
+    constructor(StrippedTableConfig: Partial<TableOptions>) {
+        super(StrippedTableConfig);
+    }
+
+    public toString() {
+        return colors.strip(super.toString());
+    }
+}
+
+/**
+ * @typedef {Object} AssemblerConfig
```

Comment:
Define an interface instead of the JSDocs

Reply:
As discussed used the one defined in index.ts